packages/xo-web/src/xo-app/index.js

```diff
@@ -114,6 +116,33 @@ const BODY_STYLE = {
     signedUp: !!state.user,
   }
 })
+@provideState({
+  initialState: () => ({ checkXoaCount: 0 }),
+  effects: {
+    async forceRefreshXoaStatus() {
+      await clearXoaCheckCache()
+      await this.effects.refreshXoaStatus()
+    },
+    refreshXoaStatus() {
+      this.state.checkXoaCount += 1
+    },
+  },
+  computed: {
+    xoaStatus: {
+      get({ checkXoaCount }) {
+        // To avoid aggressive minification which would remove destructuration
+        noop(checkXoaCount)
+        return getXoaPlan() === 'Community'
+          ? ''
+          : checkXoa().catch(() => {
+              throw new Error('XOA Plugin not registered')
+            })
```

Comment:
What's the expected behavior by throwing here?

Reply:
If the promise `checkXoa` has been rejected we are throwing an `error` in the web console.

Reply:
Thanks for answering, I'll let you see this with @pdonias :slightly_smiling_face:

Reply:
I'm not sure it's a good idea to have a computed throw an error since the error can't be handled. So I'd suggest that we simply return `''` if the promise rejects.